pkg/addon/policyframework/manifests/managedclusterchart/templates/cleanup_pod.yaml

```diff
@@ -11,8 +11,10 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
     addon.open-cluster-management.io/hosted-manifest-location: hosting
+  {{- if (not .Values.onMulticlusterHub) }}
```

Comment:
Maybe you want this whole pod in this `if`? Right now, I think this makes it always try to be deleting everything when it's on the hub.

Reply:
Haha, indeed 🤣 
```
list size doesn't match, expected 1 actual 2
```